### ISSUE TITLE
How is HyperSched related to AdaptDL?

### ISSUE THREAD

**Issue by anovv** [ucbrise/hypersched#4]
Hi, 

I've recently come across [AdaptDL](https://adaptdl.readthedocs.io/en/latest/) - a hyperparameter/GPU resources aware scheduler for Deep Learning training which dynamically adjusts batch size and learning rate as well as GPU budget per job for the most optimal outcome (they use custom metric called 'goodput' based on heuristics and their custom compute/network latency model). Curious how it's related to HyperSched and if there is an overlap?

Thanks!
